.github/workflows/production-suite.yml

```diff
@@ -77,3 +77,90 @@ jobs:
         with:
           name: production-tarball
           path: /tmp/production-build
+
+  production_install:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - docker_image: mepriyank/actions:bionic
+            name: Bionic Production Install
+            is_bionic: true
+            os: bionic
+
+          - docker_image: mepriyank/actions:focal
+            name: Focal Production Install
+            is_focal: true
+            os: focal
+
+    name: ${{ matrix.name  }}
+    container: ${{ matrix.docker_image }}
+    runs-on: ubuntu-latest
+    needs: production_build
+    env:
+      # GitHub Actions overrides HOME to /github/home/ but we want
+      # undo the override so it points to /home/github.
+      HOME: /home/zulip
```

Comment:
These two lines disagree about it being `/home/zulip` or `/home/github`.  Below, we're using `/home/github/zulip`, so I assume `/home/github` is right?

Also, the question this comment leaves me with is why the `$HOME` needs to be overridden.  What happens if it isn't?

Reply:
I don't remember exactly why I added this (I did the initial working version over a week or two ago)! This should be required though because if this was required some step would have failed by now. I'll remove this and test again.

For additional context, without this the psql command fails because it looks for `/home/github/.pgpass`  while we write it to `~/.pgpass` which is `/github/home/.pgpass`.

Reply:
This was not needed, and test passed successfully after removing this. Thanks @alexmv for noticing this.